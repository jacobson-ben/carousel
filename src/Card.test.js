import React from "react";
import { render, fireEvent } from "@testing-library/react";
import Card from "./Card";

it("renders without crashing", function() {
  render(<Card caption="Dog" src="https://img.webmd.com/dtmcms/live/webmd/consumer_assets/site_images/article_thumbnails/other/dog_cool_summer_slideshow/1800x1200_dog_cool_summer_other.jpg" totalNum={3} currNum={1}/>)
})

it("matches snapshot", function() {
  const { container } = render(<Card caption="Dog" src="https://img.webmd.com/dtmcms/live/webmd/consumer_assets/site_images/article_thumbnails/other/dog_cool_summer_slideshow/1800x1200_dog_cool_summer_other.jpg" totalNum={3} currNum={1}/>)
  expect(container).toMatchSnapshot();
})