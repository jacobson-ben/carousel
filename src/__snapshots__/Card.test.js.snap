// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      Dog
    </h4>
    <img
      alt="Dog"
      class="Card-image"
      src="https://img.webmd.com/dtmcms/live/webmd/consumer_assets/site_images/article_thumbnails/other/dog_cool_summer_slideshow/1800x1200_dog_cool_summer_other.jpg"
    />
    <small
      class="Card-small"
    >
      Image 
      1
       of 
      3
      .
    </small>
  </div>
</div>
`;
